cmd/minishift/cmd/config/view.go

```diff
@@ -63,35 +72,38 @@ func init() {
 	ConfigCmd.AddCommand(configViewCmd)
 }
 
-func configView() error {
-	cfg, err := ReadConfig()
+func determineTemplate(tempFormat string) (tmpl *template.Template) {
+	if tempFormat != DefaultConfigViewFormat {
```

Comment:
This is not needed. The problem why the "\n" was needed was due to the use of the buffer and the buffer not getting synced without new lines. Also by using the buffer we needed to split the string afterwards again, which could be problematic.